src/software/jetson_nano/display/screens/chip_and_kick_screen.py

```diff
@@ -0,0 +1,66 @@
+from screen import Screen
+
+ENABLE_INDEX = 0
+CHIP_INDEX = 1
+KICK_INDEX = 2
+
+PADDING = 6
+BASE_Y = 20
+
+
+class ChipAndKickScreen(Screen):
+    """
+    This screen is used to edit the chip and kick speed settings
+    """
+
+    def __init__(self, lcd_display, redis_dict, screen_actions):
+        """
+        @param lcd_display, an instance of the LcdDisplay class
+        @param redis_dict, a dict of values from redis client to init variables on this screen
+        @param screen_actions, an instance of ScreenActions class
+        """
+        actions = [
+            {
+                "redis key": "chip and kick enable",
+                "value": redis_dict["chip and kick enable"],
+                "type": bool,
+                "delta": None,
+                "screen action": screen_actions.UPDATE_REDIS,
+                "display string": "Set Chip & Kick: ",
+            },
```

Comment:
lets seperate this out to
* chip
* kick

because if people set both kick and chip, depending on which one they select the chip/kick will happen


if we set chip or  kick back to false in redis, will the LCD reflect that?

Reply:
I have seperated chip and kick. 

Yes, if chip or kick is set to false in redis, LCD will reflect that within a few seconds.